src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -53,12 +54,6 @@ export default function Drawers() {
   );
 }
 
-/**
- * The ActiveDrawer component will render either the drawer content that corresponds
- * to the activeDrawerId or the Tools content if it exists and isToolsOpen is true.
- * The aria labels, click handling, and focus handling will be updated dynamically
- * based on the active drawer or tools content.
- */
```

Comment:
Removed the comment because it is no longer accurate after removing dependency on tools props